app/views/layouts/_navigation.html.erb

```diff
@@ -1,21 +1,21 @@
 <nav class="navbar navbar-inverse p-0 position-absolute justify-content-between w-100">
-  <div class="container-fluid p-0">
-    <div class="navbar-header px-5 py-3">
-      <%= link_to 'Sports Blog', root_path, class: 'navbar-brand m-0' %>
+  <div class="container">
+    <div class="navbar-header">
+      <%= link_to 'Pundit Blog', root_path, class: 'navbar-brand' %>
```

Comment:
looks like unnecessary change from `Sports` to `Pundit`

Reply:
it is typo

Reply:
solved